waku/v2/protocol/waku_store/waku_store.nim

```diff
@@ -47,8 +47,8 @@ proc computeIndex*(msg: WakuMessage): Index =
   ctx.update(msg.payload)
   let digest = ctx.finish() # computes the hash
   ctx.clear()
-  result.digest = digest
-  result.receivedTime = epochTime() # gets the unix timestamp
```

Comment:
@oskarth related to my comment about the lack of receiver timestamp computation, please see this line. In here we generate the receiver-side timestamp as part of the Index struct, but in this PR, this is going to be replaced by the sender timestamp,